src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -170,13 +169,6 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         // name: /Thursday, January 28, 2021/,
       }),
     ).to.be.ok;
-
-    // Verify breadcrumb links works...
-    const VAOSHomepageLink = await screen.findByRole('link', {
```

Comment:
We already click on this link once in this test, it doesn't seem strictly necessary to do it again, and for some reason this section of the test is flaky.